The release runtime should suffice for -stage0, the release
and debug runtimes will be built in community/ldc as usual.

--- a/ldc/runtime/CMakeLists.txt
+++ b/ldc/runtime/CMakeLists.txt
@@ -818,17 +818,6 @@ macro(build_runtime_variants d_flags c_flags ld_flags path_suffix outlist_target
         "${COMPILE_ALL_D_FILES_AT_ONCE}"
         ${outlist_targets}
     )
-    # static and/or shared debug druntime/Phobos
-    build_runtime_variant(
-        "${d_flags};${D_FLAGS};${D_FLAGS_DEBUG}"
-        "${c_flags}"
-        "${ld_flags}"
-        "-debug"
-        "${path_suffix}"
-        "OFF" # no additional bitcode/LTO objects
-        "${COMPILE_ALL_D_FILES_AT_ONCE}"
-        ${outlist_targets}
-    )
 endmacro()
 
 # Setup the build of jit runtime
@@ -1068,7 +1057,6 @@ endfunction()
 # Generates targets for static and/or shared debug+release druntime/Phobos test runners.
 function(build_test_runner_variants path_suffix d_flags c_flags)
     build_test_runner_variant("" "${path_suffix}" "${D_FLAGS_RELEASE};${d_flags}" "${c_flags}")
-    build_test_runner_variant("-debug" "${path_suffix}" "${D_FLAGS_DEBUG};${d_flags}" "${c_flags}")
 endfunction()
 
 set(TESTLIB_SUFFIX "${LIB_SUFFIX}")
@@ -1130,11 +1118,9 @@ endfunction()
 function(add_runtime_tests_variants path_suffix)
     if(NOT ${BUILD_SHARED_LIBS} STREQUAL "ON")
         add_runtime_tests("" "${path_suffix}")
-        add_runtime_tests("-debug" "${path_suffix}")
     endif()
     if(NOT ${BUILD_SHARED_LIBS} STREQUAL "OFF")
         add_runtime_tests("${SHARED_LIB_SUFFIX}" "${path_suffix}")
-        add_runtime_tests("-debug${SHARED_LIB_SUFFIX}" "${path_suffix}")
     endif()
 endfunction()
 
@@ -1143,4 +1129,3 @@ if(MULTILIB AND ${HOST_BITNESS} EQUAL 64)
     add_runtime_tests_variants("${MULTILIB_SUFFIX}")
 endif()
 
-include(DRuntimeIntegrationTests)
